### ISSUE TITLE
Yolo test, train sets

### ISSUE THREAD

**Issue by saroshpatel** [ultralytics/ultralytics#16969]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello,

A yolo custom training dataset requires 3 folders train, test and valid

I am a little confused as when the Train and Valid sets are used by the model during training ? And the difference between them

Thank you !

**Comment by glenn-jocher** [comment]
@saroshpatel the train set is used to fit the model, while the valid set evaluates its performance during training to prevent overfitting. The test set is for final evaluation after training. For more details, visit the [Ultralytics documentation](https://docs.ultralytics.com/datasets/classify/).

**Comment by saroshpatel** [comment]
Thank you for your answer !

So at the end of training when I validate the model using model.val( ), do I need to insist on model.val( split ='test') ?

**Comment by glenn-jocher** [comment]
@saroshpatel you're welcome! Yes, using `model.val(split='test')` will evaluate the model on the test set specifically.
